### PR TITLE
Added a PSC GCLB example

### PR DESCRIPTION
**Pull request by cgrotz**
I created a simple blueprint where a L7XLB is connecting to an application in another project via a PSC Neg. The application in turn is exposed internally via a L7ILB, that is exposed via PSC. The application is a Cloud Run service, but could be anything really.

### CODE REVIEW

**Comment by ludoo** [comment]
Can you add a basic test like other blueprints? Otherwise it will stop working as soon as we change one of its modules...

**Comment by cgrotz** [comment]
Just discovered the tests, will add one thank you. Can you give me some guidance what is missing for the linting check?

**Comment by ludoo** [comment]
> Just discovered the tests, will add one thank you. Can you give me some guidance what is missing for the linting check?

wanna have a quick vc?

**Comment by ludoo** [comment]
All you need is to run tfdoc on the blueprint, as explained in the contributing guide. Ping me if you want to do it together.

**Comment by cgrotz** [comment]
I'll try to figure it out, I will ping you if I can't make it work.

File: blueprints/networking/psc-glb-and-armor/variables.tf [hunk 1]
```diff
@@ -14,11 +14,35 @@
...
+  description = "The consumer project, in which the GCLB and Cloud Armor should be created."
 }
+
 variable "producer_project_id" {
+  description = "The producer project, in which the ILB, PSC Service Attachment and Cloud Run service should be created"
 }
 
 variable "region" {
   default = "europe-west1"
+  description = "The GCP region in which the resources should be deployed."
```
Comment (ludoo) [inline blueprints/networking/psc-glb-and-armor/variables.tf:42]: nit: can you put description before default, here and below?

Comment (ludoo) [reply blueprints/networking/psc-glb-and-armor/variables.tf:42]: and make sure a type is declared on all variables

**Review by ludoo — APPROVED** [review]

File: tests/blueprints/networking/psc-glb-and-armor/test_plan.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2022 Google LLC
```
Comment (juliocc) [inline tests/blueprints/networking/psc-glb-and-armor/test_plan.py:1]: Can you use the new style tests?

I know they're not documented (my bad) but they're much easier. Here's what you have to do: 
- Add an hcl block code at the end of the README. Similar to [this one.](https://github.com/GoogleCloudPlatform/cloud-foundation-fabric/blob/052e310418a51dc676bc03fc39d6e6dd7764040d/blueprints/apigee/hybrid-gke/README.md#test) 
- Update the tftest directive with your own module and resource counts
- Ignore the inventory part, or if you really want I can show you how to create it.

**Review by juliocc — CHANGES_REQUESTED** [review]

File: blueprints/apigee/bigquery-analytics/README.md [hunk 3]
```diff
@@ -19,7 +19,7 @@ Note: This setup only works if you are not using custom analytics.
 
 ## Running the blueprint
 
-1. Clone this repository or [open it in cloud shell](https://ssh.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https%3A%2F%2Fgithub.com%2Fterraform-google-modules%2Fcloud-foundation-fabric&cloudshell_print=cloud-shell-readme.txt&cloudshell_working_dir=blueprints%2Fapigee%2Fbigquery-analytics), then go through the following steps to create resources:
+1. Clone this repository or [open it in cloud shell](https://ssh.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https%3A%2F%2Fgithub.com%2Fterraform-google-modules%2Fcloud-foundation-fabric&cloudshell_print=cloud-shell-readme.txt&cloudshell_working_dir=blueprints%2Fnetworking%2Fpsc-glb-and-armor), then go through the following steps to create resources:
```
Comment (juliocc) [inline blueprints/apigee/bigquery-analytics/README.md:22]: I think you update this one accidentally?

**Review by juliocc — APPROVED** [review]

**Pull request merged**